pkg/minishift/openshift/version/openshift_versions.go

```diff
@@ -91,19 +92,23 @@ func PrintDownStreamVersions(output io.Writer, minSupportedVersion string) {
 }
 
 func PrintUpStreamVersions(output io.Writer, minSupportedVersion string) {
-	resp, err := getResponseBody("https://registry.hub.docker.com/v2/repositories/openshift/origin/tags/")
+	dockerRegistryUrl := "https://registry.hub.docker.com/v2/repositories/openshift/origin/tags/?page_size=1024"
```

Comment:
I am wondering whether we should v1 of the API - https://registry.hub.docker.com/v1/repositories/openshift/origin/tags. This seems to return in a more condensed format what we want and it is proper v1 API syntax. According to the docs there should be a _tags/list_ - https://docs.docker.com/registry/spec/api/#tags - which does not seem to work. _page_size_ does not seem to be documented anywhere. I am a bit worried that this will change/break.